penny_u_django/home/static/home/style.css

```diff
@@ -0,0 +1,38 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto+Slab:700&display=swap');
+
+
+h1,h2,h3,h4,h5,.navbar-brand {
+    font-family: 'Roboto Slab', serif;
+}
+
+
+.bg-dark,.navbar-dark {
+    background-color: #595348 !important;
+}
+
+.bg-white {
+    color: #292621;
+}
+
+
+#nav-join-btn {
```

Comment:
I think we should stick to only using classes in our css, since they can appear multiple times in a page. Also, I think you can just use bootstrap utilities for this kind of thing. Check out https://getbootstrap.com/docs/4.3/utilities/spacing/

Reply:
I recommend only using ID's for things that are used once per page that tie to javascript only as it is microseconds faster supposedly.